src/devtools/client/debugger/src/components/Editor/LineHitCounts.tsx

```diff
@@ -22,8 +24,21 @@ function LineHitCounts({ cm }: Props) {
     () => (hitCounts !== null ? hitCountsToMap(hitCounts) : null),
     [hitCounts]
   );
-
   const dispatch = useAppDispatch();
+  const uniqueHitCounts = Array.from(
+    new Set(hitCounts ? hitCounts.map(hitCount => hitCount.hits) : [])
+  );
+  const maxHitCount = Math.max(...uniqueHitCounts);
+  const hitCountColorMap = useMemo(
+    () =>
+      new Map(
+        uniqueHitCounts.map(hitCount => [
+          hitCount,
+          Math.min(1, Math.max(0, Math.log(hitCount) / Math.log(maxHitCount))) || 0,
+        ])
+      ),
+    [uniqueHitCounts, maxHitCount]
+  );
```

Comment:
This memoization looks broken. We're always re-creating `uniqueHitCounts` so `useMemo` will always be bypassed. (Need to move the `Set` creation into the memo function too.)

Reply:
Oops, yeah, I'll fix this up!